include/SANDTrackerCell.h

```diff
@@ -0,0 +1,109 @@
+#pragma once
+
+#include "SANDWireInfo.h"
+
+class SANDTrackerPlane;
+
+class SANDTrackerCellID : public SingleElStruct<unsigned long>
+{
+ public:
+  SANDTrackerCellID(unsigned long id) : SingleElStruct<unsigned long>(id){};
+  SANDTrackerCellID() : SingleElStruct<unsigned long>(){};
+};
+
+class SANDTrackerCell
+{
+  SANDTrackerCellID _id;
+  SANDWireInfo _wire;
+  double _width;
+  double _height;
+  double _driftVelocity;  // drift velocity in um/ns
+
+  SANDTrackerPlane* _plane;
+  std::vector<SANDTrackerCell*> _adjacent_cells;
+
+ public:
+  SANDTrackerCell() {};
+  SANDTrackerCell(const SANDTrackerCellID cID,
+                  const SANDWireInfo &l, 
+                  const double w,
+                  const double h,
+                  const double v,
+                  SANDTrackerPlane* plane)
+      : _id(cID),
+        _wire(l),
+        _width(w),
+        _height(h),
+        _driftVelocity(v),
+        _plane(plane)
+  {
+  }
+  SANDTrackerCell(const SANDTrackerCellID cID,
+                  const SANDWireInfo &l, 
+                  const double w,
+                  const double h,
+                  const double v)
+      : _id(cID),
+        _wire(l),
+        _width(w),
+        _height(h),
+        _driftVelocity(v)
+  {
+  }
+
+  SANDTrackerCell(const SANDTrackerCellID cID, const SANDWireInfo &l, SANDTrackerPlane* plane): 
+        _id(cID), _wire(l), _plane(plane)
+  {
+  }
+
+  SANDTrackerCell(const SANDTrackerCell &cell);
+
+  void id(SANDTrackerCellID wID)
+  {
+    _id = wID;
+  }
+  void setPlane(SANDTrackerPlane* p) 
+  {
+    _plane = p;
+  }
+  SANDTrackerPlane* getPlane() const 
+  {
+    return _plane;
+  }
+  SANDTrackerCellID id() const
+  {
+    return _id;
+  }
+  void size(double &h, double &w) const
```

Comment:
To return multiple “out” values, prefer returning a struct.

https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#f21-to-return-multiple-out-values-prefer-returning-a-struct